src/table/components/footer/FooterWrapper.tsx

```diff
@@ -4,12 +4,14 @@ import { StyledFooterWrapper } from './styles';
 import { FooterWrapperProps } from '../../types';
 import { getFooterStyle } from '../../utils/styling-utils';
 
-export default function FooterWrapper({ children, theme, footerContainer }: FooterWrapperProps) {
+export default function FooterWrapper({ children, theme, footerContainer, withoutBorders }: FooterWrapperProps) {
   const footerStyle = useMemo(() => getFooterStyle(theme.background), [theme]);
 
   return footerContainer ? (
     ReactDOM.createPortal(children, footerContainer)
   ) : (
-    <StyledFooterWrapper footerStyle={footerStyle}>{children}</StyledFooterWrapper>
+    <StyledFooterWrapper footerStyle={footerStyle} withoutBorders={withoutBorders}>
```

Comment:
Virtual table does not have borders around the footer so had to make them optional

Reply:
but is this still the case? I think it should always have a top border now, for both pagination and vs

Reply:
No we (you) should change this as you say